CMSgov/mdct-mcr
# MDCT-1826: Read-Only Role (Landing Page + Dashboard)

## Description

Implementation of a read-only role for users with `Help Desk`, `CMS Approver`, and `BOR` roles in MCR. 

Closes [MDCT-1826](https://qmacbis.atlassian.net/browse/MDCT-1826).

### How to test

- ./dev local
- Login as `adminuser@test.com` 

You should only see a cute little dropdown with all 50 states (and DC + PR).

### Changed Dependencies

N/A

## Code author checklist

## Reviewer checklist (two different people)
